### ISSUE TITLE
Align segment start (and end?) times to keyframes

### ISSUE THREAD

**Issue by mifi** [mifi/lossless-cut#1356]
- Either have a keyboard shortcut / menu action that will align all segment start (and end?) times to the nearest keyframe (or the previous keyframe)
- Or a toggle setting which will decide whether or not all setCutStart operations shall align to the nearest keyframe

Kind of related #330

**Comment by kibbby** [comment]
Just following up on this. For me personally it’s a must. I’d love for it to be a toggle 

“Or a toggle setting which will decide whether or not all setCutStart operations shall align to the nearest keyframe”

I have sent several emails today about my issue with this. Did a lot of testing and YouTube doesn’t seem to like when it does this in some videos/segments in export. No idea why, it’s a bit random. Some videos are fine the whole way through and others start playing in what looks like 30fps type viewing. This happens when using random segment as it doesn’t select start and end keyframes. So I reuploaded the same video only this time I edited it manually in LC with proper start and end keyframes for each segment with success. Not a long term solution so hoping to see this in next update. Very important to me. 

Thanks!
